warden/integration-tests/action.yml

```
@@ -0,0 +1,182 @@
+name: "Integration Tests Mage-OS in Warden"
+author: "Vladyslav Podorozhnyi"
```

```suggestion
author: "Mage-OS"
```

Should we remove this or make this Mage-OS? :see_no_evil:

---

You dare... 😁